src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/RecallValidationService.kt

```diff
@@ -29,6 +29,7 @@ class RecallValidationService(
     val (recallLength, has14DayFTRSentence, has28DayFTRSentence) = getFtrValidationDetails(ftrDetails, sourceData.sentenceAndOffences)
 
     return when {
+      sourceData.returnToCustodyDate == null -> listOf(ValidationMessage(ValidationCode.FTR_NO_RETURN_TO_CUSTODY_DATE))
```

Comment:
I think you need (has14DayFTRSentence || has28DayFTRSentence) &&  sourceData.returnToCustodyDate == null